verticapy/learn/tools.py

```diff
@@ -0,0 +1,611 @@
+                        model_save["tol"],
+                        model_save["max_iter"],
+                        model_save["solver"],
+                        model_save["max_pik"],
+                        model_save["papprox_ma"],
+                    )
+                    model.transform_relation = model_save["transform_relation"]
+                    model.coef_ = tablesample(model_save["coef"])
+                    model.ma_avg_ = model_save["ma_avg"]
+                    if isinstance(model_save["ma_piq"], dict):
+                        model.ma_piq_ = tablesample(model_save["ma_piq"])
+                    else:
+                        model.ma_piq_ = None
+                    model.ts = model_save["ts"]
+                    model.exogenous = model_save["exogenous"]
+                    model.deploy_predict_ = model_save["deploy_predict"]
+                elif model_save["type"] == "VAR":
+                    from verticapy.learn.tsa import VAR
+
+                    model = VAR(
+                        name,
+                        cursor,
+                        model_save["p"],
+                        model_save["tol"],
+                        model_save["max_iter"],
+                        model_save["solver"],
+                    )
+                    model.transform_relation = model_save["transform_relation"]
+                    model.coef_ = []
+                    for i in range(len(model_save["X"])):
+                        model.coef_ += [tablesample(model_save["coef_{}".format(i)])]
+                    model.ts = model_save["ts"]
+                    model.deploy_predict_ = model_save["deploy_predict"]
+                    model.X = model_save["X"]
+                model.input_relation = model_save["input_relation"]
+                model.X = model_save["X"]
+                if model_save["type"] in (
+                    "KNeighborsRegressor",
+                    "KNeighborsClassifier",
+                    "NearestCentroid",
+                    "SARIMAX",
+                ):
+                    model.y = model_save["y"]
+                    model.test_relation = model_save["test_relation"]
+                elif model_save["type"] not in ("CountVectorizer", "VAR"):
+                    model.key_columns = model_save["key_columns"]
+                return model
+    try:
+        info = info.split("SELECT ")[1].split("(")
+    except:
+        info = info.split("(")
+    model_type = info[0].lower()
+    info = info[1].split(")")[0].replace(" ", "").split("USINGPARAMETERS")
+    if model_type == "svm_classifier":
+        parameters = "".join(info[1].split("class_weights=")[1].split("'"))
+        parameters = parameters[3 : len(parameters)].split(",")
+        del parameters[0]
+        parameters += [
+            "class_weights=" + info[1].split("class_weights=")[1].split("'")[1]
+        ]
+    elif model_type != "svd":
+        parameters = info[1].split(",")
+    if model_type != "svd":
```

Comment:
Why the if condition here is similar to the condition of elif on line 323?

Reply:
Corrected;